lib/grandpa/vote_message.go

```diff
@@ -36,6 +38,33 @@ func (s *Service) receiveMessages(ctx context.Context) {
 			logger.Tracef("received vote message %v from %s", msg.msg, msg.from)
 			vm := msg.msg
 
+			switch vm.Message.Stage {
+			case prevote:
+				err := telemetry.GetInstance().SendMessage(
+					telemetry.NewAfgReceivedPrevoteTM(
+						vm.Message.Hash,
+						fmt.Sprintf("%d", vm.Message.Number),
+						vm.Message.AuthorityID.String(),
+					),
+				)
+				if err != nil {
+					logger.Debugf("problem sending afg.received_prevote telemetry message, err: %s", err)
```

Comment:
nit no need for `, err:`. Same for other log calls below

```suggestion
					logger.Debugf("problem sending afg.received_prevote telemetry message: %s", err)
```